internal/config/config.go

```diff
@@ -65,25 +65,29 @@ func ClientConfigFile() string {
 	return filepath.Join(ConfigDir(), "client.yaml")
 }
 
-func NewDefault() *Config {
+func NewDefault() (*Config, error) {
+  port, err := util.GetIntEnv("db.port", 5432)
+  if err != nil {
+    return nil, err
+  }
 	c := &Config{
 		Database: &dbConfig{
 			Type:     "pgsql",
-			Hostname: "localhost",
-			Port:     5432,
-			Name:     "planner",
-			User:     "admin",
-			Password: "adminpass",
+			Hostname: util.GetEnv("db.host", "localhost"),
```

Comment:
Actually dot in a env name is invalid. We need to rename it to something like `DB_HOST`